src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -287,29 +287,31 @@ public void onMessage(String channel, String message) {
                 if (masterName.equals(switchMasterMsg[0])) {
                   initPool(toHostAndPort(Arrays.asList(switchMasterMsg[3], switchMasterMsg[4])));
                 } else {
-                  log.fine("Ignoring message on +switch-master for master name "
-                      + switchMasterMsg[0] + ", our master name is " + masterName);
+                  log.debug(
+                    "Ignoring message on +switch-master for master name {}, our master name is {}",
+                    switchMasterMsg[0], masterName);
                 }
 
               } else {
-                log.severe("Invalid message received on Sentinel " + host + ":" + port
-                    + " on channel +switch-master: " + message);
+                log.error(
+                  "Invalid message received on Sentinel {}:{} on channel +switch-master: {}", host,
+                  port, message);
               }
             }
           }, "+switch-master");
 
         } catch (JedisException e) {
 
           if (running.get()) {
-            log.log(Level.SEVERE, "Lost connection to Sentinel at " + host + ":" + port
-                + ". Sleeping 5000ms and retrying.", e);
+            log.error("Lost connection to Sentinel at {}:{}. Sleeping 5000ms and retrying.", host,
```

Comment:
I guess `e` is treated to one of Object varargs, not treated to Throwable object. @r0main Could you check it?
If I'm right, you need to use traditional string concatenation or String.format() in this case.